common-content-dataflow/modules/ROOT/partials/changelog.adoc

```diff
@@ -0,0 +1,101 @@
+= Changelog
+
+This page lists changes to the Dataflow Flex Template for BigQuery and/or Google Cloud to Neo4j.
+
+You can check the deployment status of a release by running `gsutil ls gs://dataflow-templates/`.
+
+Releases are gradually deployed, some regions may be updated before others.
+To check the release status of https://cloud.google.com/compute/docs/regions-zones[a particular region], you can run `gsutil ls gs://dataflow-templates-$\{region}/` (e.g. `gsutil ls gs://dataflow-templates-europe-west2/`).
+
+== Release 2025-06-10-01_RC00
+
+* Add support to see underlying Neo4j driver's Bolt logs
+
+== Release 2025-01-26-00_RC00
+
+* Add support for calendar-based version names of the Neo4j database
+
+== Release 2025-01-22-01_RC00
+
+* Switch to Java 17
+* Updated Neo4j driver version from v4.4.18 to v5.27.0
+
+== Release 2025-01-14-00_RC00
+
+* Re-added node target's existence constraint
+
+== Release 2025-01-07-00_RC01
+
+* Remove node target's existence constraint
+* Fix an issue with custom Cypher queries
```

Comment:
I find these sort of entries quite useless from a user's perspective. Is it a very-hard-to-describe issue, or can we at least hint at what it affected?

Reply:
Did some git history research, read some code. Posting an update to what changed.